contracts/Ovm_SpokePool.sol

```diff
@@ -137,7 +147,9 @@ contract Ovm_SpokePool is SpokePool {
         SpokePoolInterface.RelayerRefundLeaf memory relayerRefundLeaf,
         bytes32[] memory proof
     ) public override(SpokePool) nonReentrant {
-        if (relayerRefundLeaf.l2TokenAddress == address(wrappedNativeToken)) _depositEthToWeth();
+        // Leave any ETH we need to send back to L1 unwrapped.
+        if (relayerRefundLeaf.l2TokenAddress == address(wrappedNativeToken))
```

Comment:
What if there isn't enough unwrapped ETH to cover it? For instance, if we have 10 deposits of 1 ETH, they would be auto-converted to WETH. If the amountToReturn were > 0, this would fail, right? We would need some way to _unwrap_ the WETH, right?

Should we set this up so that it wraps _or_ unwraps to achieve a specific amount of ETH that we want (which would _usually_ be 0)?

Reply:
Yeah I'm gonna revert this change. The issue I was trying to address is that in the Optimism_SpokePool unit test, we don't actually unwrap any WETH because WETH is a faked contract. Therefore the payable call to `withdrawTo` reverts in the tests. I think one way around this is to make WETH a real contract

Reply:
Gotcha, makes sense!